### PR TITLE
RRCP archived tests - show tests in campaigns page

### PR DESCRIPTION
**Pull request by KaliedaRik**
## What does this change?
Task: Display archived tests in RRCP through the Campaigns page

This PR makes the following changes to the RRCP Campaigns page. Note that it does **not** cover the work to display an archived test's _form data_, which will need to be taken forward in a separate PR (to keep the size of this PR manageable).

- Adds a new button in the campaign listing sticky bar to toggle the display of archived tests in the listing
- Amends the display of test cards so that tests with a status of `live` or `Draft` outline their card with a solid outline, while `Archived` test cards display with a dashed outline
- Alphabetically sorts the left hand list of campaigns
- Adds a new button to the bottom of the left hand list of campaigns which, when clicked, will display a listing of all tests not associated with any campaign
- Updates the `Update Test status values` button's modal so that archived tests are excluded from the toggle list of tests, and the tests that are listed are ordered first by channel and then alphabetically

### Screenshots: before changes
Screenshot 2022-07-15 at 15 01 17

Screenshot 2022-07-15 at 15 01 33

### Screenshots: after changes
Screenshot 2022-07-15 at 16 00 47

Screenshot 2022-07-15 at 16 01 07

Screenshot 2022-07-15 at 16 01 22

Screenshot 2022-07-15 at 16 01 53

Screenshot 2022-07-15 at 16 02 09

### CODE REVIEW

File: public/src/components/channelManagement/campaigns/CampaignsList.tsx [hunk 1]
```diff
@@ -56,20 +56,47 @@ const CampaignsList = ({
...
+        {sortedCampaigns.map(campaign => (
           <ListItem className={classes.listItem} key={campaign.name}>
             <Button
               className={classes.button}
               variant={checkIfCampaignIsSelected()}
               onClick={(): void => onCampaignSelected(campaign.name)}
             >
-              <Typography className={classes.text}>{campaign.nickname}</Typography>
+              <Typography className={classes.text}>
+                {campaign.nickname != null ? campaign.nickname : campaign.name}
```
Comment (MaelGNM) [inline public/src/components/channelManagement/campaigns/CampaignsList.tsx:86]: Could nickname be empty string? In that case we would want to see the name instead

Comment (KaliedaRik) [reply public/src/components/channelManagement/campaigns/CampaignsList.tsx:86]: Good call. Will update.

File: public/src/components/channelManagement/campaigns/CampaignsForm.tsx [hunk 2]
```diff
@@ -53,6 +54,12 @@ export interface Campaign {
 }
 export type Campaigns = Campaign[];
 
+export const unassignedCampaign = {
+  name: unassignedCampaignLabel,
+  nickname: 'TESTS NOT IN A CAMPAIGN',
```
Comment (tomrf1) [inline public/src/components/channelManagement/campaigns/CampaignsForm.tsx:59]: this list of tests might get very large! Something to keep an eye on

Comment (KaliedaRik) [reply public/src/components/channelManagement/campaigns/CampaignsForm.tsx:59]: Agreed. I want this to be a temporary fix that we can rip out once we have added search functionality to RRCP

**Pull request merged**